src/js/edu-benefits/1990e/reducer.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
```

Comment:
reducer.js and routes.jsx are identical for the 1995 and 1990e, so do we want to move these into src/js/common?